pkg/spdk/server.go

```
@@ -236,23 +251,55 @@ func (s *Server) verify() (err error) {
 			if replicaMap[GetReplicaNameFromRebuildingLvolName(lvolName)] != nil {
 				continue
 			}
 		}
-		lvsUUID := bdevLvol.DriverSpecific.Lvol.LvolStoreUUID
-		specSize := bdevLvol.NumBlocks * uint64(bdevLvol.BlockSize)
-		actualSize := bdevLvol.DriverSpecific.Lvol.NumAllocatedClusters * uint64(defaultClusterSize)
-		replicaMap[lvolName] = NewReplica(s.ctx, lvolName, lvsUUIDNameMap[lvsUUID], lvsUUID, specSize, actualSize, s.updateChs[types.InstanceTypeReplica])
-		replicaMapForSync[lvolName] = replicaMap[lvolName]
+		if IsBackingImageSnapLvolName(lvolName) {
+			lvsUUID := bdevLvol.DriverSpecific.Lvol.LvolStoreUUID
+			backingImageName := GetBackingImageNameFromLvolName(lvolName)
```

golangci-lint (1.62.2)

ineffectual assignment to backingImageName

(ineffassign)